routes/error_messages_factory.js

```diff
@@ -0,0 +1,3 @@
+module.exports.errorMessageToJson = (err, text) => {
```

Comment:
- Name files in camelCase
- Chose between `errorMessages` or `errorMessage` (I would prefer singular)